src/actors.rs

```diff
@@ -41,6 +41,11 @@ pub async fn new_actor(
     new_actor: web::Json<NewActor>,
 ) -> impl Responder {
     let name = new_actor.into_inner().name;
+    let name_length = name.chars().count();
+    // Early response for if the name is too long
```

Comment:
nit: I'm not sure this adds more information about the following code. If there's going to be a comment, maybe focus on why the length is limited to 0-30?